linux/debian/rules

```diff
@@ -62,7 +59,7 @@ override_dh_auto_install:
 	install -d $(CURDIR)/debian/tmp
 	cd core && DESTDIR=$(CURDIR)/debian/tmp ./build.sh install
 	# ibus-keyman
-	cd linux/ibus-keyman && make install DESTDIR=$(CURDIR)/debian/tmp
+	cd linux/ibus-keyman && DESTDIR=$(CURDIR)/debian/tmp ./build.sh install
```

Comment:
Is it still appropriate to be setting `DESTDIR` here?

Reply:
yes, that's needed so that the files we install end up in a dedicated directory so that the packaging system knows what to put in the package.